Cargo.toml

```diff
@@ -25,6 +25,7 @@ native-tls = "0.2.3"
 lettre = "0.9.0"
 imap = "1.0.1"
 # rental = "0.5.3"
+mmime = { path = "../mmime" }
```

Comment:
Aaah, so `mmime` needs to be checked out locally for now?

Reply:
just fixed